.travis.yml

```diff
@@ -34,4 +27,31 @@ jobs:
       script:
         - "$HOME/travisci-tools/fsc-trigger/trigger_fullstack-sdk-compat.sh"
       after_success: travis_terminate 0
-      
\ No newline at end of file
+    - &test
+      stage: 'Unit test'
+      env: GIMME_GO_VERSION=master GIMME_OS=linux GIMME_ARCH=amd64
+      merge_mode: merge
+      script:
+        - go test -v -race ./... -coverprofile=profile.cov
+    - <<: *test
+      stage: 'Unit test'
+      merge_mode: merge
+      env: GIMME_GO_VERSION=1.8.x
+      before_script:
+        # GO module was not introduced earlier. need symlink to search in GOPATH
+        - mkdir -p $GOPATH/src/github.com && pushd $GOPATH/src/github.com && ln -s $HOME/build/optimizely optimizely && popd
+      script:
+        # Need to download packages explicitly
+        - go get -v -d ./...
```

Comment:
I think we can just put these in a script inside of `scripts` so that our travis file is simpler

Reply:
it's actually only for go1.8, that's why I didn't create any file.